packages/grid/src/features/standard-table/hooks/UseGridCellMemo.ts

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from "react";
+import {
+  useGridCell,
+  UseGridCellOptions,
+  UseGridCellResult
+} from "../../grid-cell/hooks/UseGridCell";
+
+export const useGridCellMemo = <TValue>(
```

Comment:
isn't gridCell already wrapped with `useMemo`?

Reply:
No, not all fields. I'll fix it.

Reply:
Removed useGridCellMemo.